libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -464,6 +472,30 @@ bool AP_BattMonitor::arming_checks(size_t buflen, char *buffer) const
     return true;
 }
 
+// Check's each smart battery instance for its powering off state and broadcasts notifications
+void AP_BattMonitor::checkPoweringOff(void)
+{
+    static bool powerOffNotified[AP_BATT_MONITOR_MAX_INSTANCES];
+
+    for (uint8_t i = 0; i < _num_instances; i++) {
+        if (is_powering_off(i) && !powerOffNotified[i]) {
+            // Set the AP_Notify flag, which plays the power off tones
+            AP_Notify::flags.powering_off = true;
```

Comment:
The entire powering off flag can now be removed and instead use the helper function. If you don't want to use the helper function in notify then the notify flag can be used here rather then the static member. (Basically I'm just objecting to 2 different ways to get to the same information about the battery library).

Reply:
Removed the unnecessary bool, just checking the state directly.